src/components/dataDisplays/DataDisplay.jsx

```diff
@@ -61,11 +58,9 @@ export default function DataDisplay({
   tableSize = 'small',
   noTitleBar,
   loading,
-  paginated = false,
-  paginatedExternally = true, // display all data provided and let parent component(s) paginate
-  page,
-  onChangePage,
-  rowsPerPage,
```

Comment:
pagination will no longer be handled within the DataDisplay component.
1 - this component is too big already 
2 - checked material-ui examples and they actually render pagination controls outside of the html <table> container
3 - want to plan for other UIs (such as the individual search gallery) which will have pagination but not tables